content/kotlin/concepts/datetime/terms/encodeToString/encodeToString.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'encodeToString()'
+
+Description: 'Serializes data to a string representation using the Kotlin Serialization library.'
+
+Subjects:
+  -'Kotlin'
+  -'Serialization'
+
+Tags:
+  -'Kotlin Serialization'
+  -'Serialization Library'
+  -'JSON Serialization'
```

Comment:
None of these [tags](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) exist here. Either add these there or add the tags that exist there.